source/github-helpers/selectors.ts

```diff
@@ -205,6 +205,16 @@ export const usernameLinksSelector = [
 	// GHE sometimes shows the full name already:
 	// https://github.com/refined-github/refined-github/issues/7232#issuecomment-1910803157
 
+	// On new issue view
+	`.react-issue-comment
+	a[data-testid='avatar-link']:not(
+		[href='#'],
+		[href*='/apps/'],
+		[href*='/marketplace/'],
+		[data-hovercard-type='organization'],
+		[show_full_name='true']
```

Comment:
I wasn't able to find anywhere to check if these still apply.

Reply:
Yes they most likely apply except for the last one. We can just keep it because we did remove it too early last time.